docs/docsite/rst/userguide/rbac.rst

```diff
@@ -0,0 +1,513 @@
+
+
+Resource access for teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section provides a reference for managing team roles within individual resources as shown in the new UI and the corresponding API calls.
+
+Access the resource's **Team Access** tab to manage the team roles.
+
+.. image:: ../common/images/rbac_jt_team_access.png
+
+To obtain a list of team role assignments from the API: 
+
+::
+
+   GET <awxAPI>/role_team_assignments/?object_id=<template_id>&content_type__model=jobtemplate
+
+The columns are arranged so that the team name appears in the first column. The role name is under ``summary_fields.role_definition.name``
+
+To revoke a role assignment for a team in the API: 
+
+::
+
+   DELETE <awxAPI>/role_team_assignments/<role_id_from_list_API_above>/
+
+
+Add roles
+^^^^^^^^^^
+
+Clicking the **Add roles** button from the **Team Access** tab opens the **Add roles** wizard, where you can select the teams to which you want to add roles.
+
+.. image:: ../common/images/rbac_team_access_add-roles.png
+
+To list the teams from the service endpoint:
+
+::
+
+   GET <awxAPI>/teams
+
+
+The next step of the wizard in the controller UI is to apply roles to the selected team(s).
+
+.. image:: ../common/images/rbac_team_access_apply-roles.png
+
+To list available role definitions for the selected resource type in the API, issue the following, but replace ``content_type`` below to match the resource type:
+
+::
+
+   GET <awxAPI>/role_definitions/?content_type__model=jobtemplate
+
+
+Finally, review your selections and click **Save** to save your changes.
+
+.. image:: ../common/images/rbac_team_access_add-roles-review.png
+
+To assign roles to selected teams in the API, you must assign a single role to individual teams separately by referencing the team ID and resource ID from the controller associated with the ``object_id`` (``jobtemplate.id`` in this example):
+
+::
+
+   POST <awxAPI>/role_team_assignments/
+
+
+A message displays to confirm the changes were successfully applied:
+
+.. image:: ../common/images/rbac_team_access_add-roles-success.png
+
+In the API, the following message displays to confirm the changes were successfully applied:
+
+::
+
+   {"team": "011-323-232", "role_definition": 4, "object_id": "123-432-233"}
+
+
+Resource access for users
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section provides a reference for managing user roles within individual resources as shown in the new UI and the corresponding API calls.
+
+Access the resource's **User Access** tab to manage the user roles.
+
+.. image:: ../common/images/rbac_jt_user_access.png
+
+To obtain a list of user role assignments from the API: 
+
+::
+
+   GET <awxAPI>/role_user_assignments/?object_id=<template_id>&content_type__model=jobtemplate
+
+The columns are arranged so that the team name appears in the first column. The role name is under ``summary_fields.role_definition.name``
```

Comment:
```suggestion
The columns are arranged so that the user name appears in the first column. The role name is under ``summary_fields.role_definition.name``
```